source/administration/changelog.md

```diff
@@ -197,6 +197,8 @@ Multiple setting options were added to `config.json`. Below is a list of the add
      - Added ``EnableSearching`` to enable search queries to use bleve search.
      - Added ``EnableAutocomplete`` to enable autocomplete queries to use bleve search.
      - Added ``BulkIndexingTimeWindowSeconds`` to determine the maximum time window for a batch of posts being indexed by the Bulk Indexer.
+ - Under ``EmailSettings`` in ``config.json``:
+     - Changed ``PushNotificationServer`` to default ``full`` for new installs.
```

Comment:
It's this setting: https://docs.mattermost.com/administration/config-settings.html#push-notification-contents